test/acceptance/features/steps/openshift.py

```diff
@@ -334,3 +335,14 @@ def search_resource_lst_in_namespace(self, resource_plural, name_pattern, namesp
             return lst
         print('Resource list is empty under namespace - {}'.format(namespace))
         return None
+
+    def get_secret_info_custom_env_var(self, secret_name, namespace):
```

Comment:
I sense this function is getting data from the JSON path and then converting it into byte objects and then decoding it. So what I would suggest instead of adding a new dedicated function for it you should use the already available generic JSON path function we have and do this byte conversion and decode in your step function. [Here's the jsonpath function you can use](https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/steps/openshift.py#L228)

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/597/commits/b904a2186b93b06ef06213f2a23139572628abdc